static/app/components/modals/inviteMembersModal/inviteStatusMessage.tsx

```diff
@@ -46,14 +41,18 @@ function CountMessage({sentCount, errorCount, isRequest}: CountMessageProps) {
     <div>
       {sentCount > 0 && (
         <StatusMessage status="success" isNewInviteModal>
-          <IconCheckmark size="sm" />
-          <span>{tct('[invites] sent.', tctComponents)}</span>
+          <IconCheckmark size="sm" color="successText" />
+          <span role="alert" aria-label={t('sent-invites')}>
```

Comment:
```suggestion
          <span role="alert" aria-label={t('Sent Invites')}>
```

The aria label is human readable